Assets/Scripts/Models/Utility.cs

```diff
@@ -0,0 +1,900 @@
+#region License
+// ====================================================
+// Project Porcupine Copyright(C) 2016 Team Porcupine
+// This program comes with ABSOLUTELY NO WARRANTY; This is free software, 
+// and you are welcome to redistribute it under certain conditions; See 
+// file LICENSE, which is part of this source code package, for details.
+// ====================================================
+#endregion
+using System;
+using System.Collections.Generic;
+using System.Xml;
+using System.Xml.Schema;
+using System.Xml.Serialization;
+using Animation;
+using MoonSharp.Interpreter;
+using MoonSharp.Interpreter.Interop;
+using ProjectPorcupine.PowerNetwork;
+using UnityEngine;
+
+/// <summary>
+/// InstalledObjects are things like walls, doors, and furniture (e.g. a sofa).
+/// </summary>
+[MoonSharpUserData]
+public class Utility : IXmlSerializable, ISelectable, IContextActionProvider, IBuildable
```

Comment:
Would it make more sense to have a Buildable base class and have Furniture and Inventory extend it? Right now both classes look almost identically. Or Utility could be a Furniture.